tox.ini

```diff
@@ -31,6 +31,14 @@ deps = -r{toxinidir}/requirements.txt
 commands =
   black {posargs} .
 
+[testenv:isort]
+basepython = python3
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+       isort
+commands =
+  isort --dont-order-by-type {posargs} {toxinidir}
```

Comment:
The isort docs are a bit confusing on this, they recommend both options for projects following PEP8:

https://pycqa.github.io/isort/docs/configuration/options/#order-by-type
> If your project follows PEP8 or a related coding standard and has many imports this is a good default, otherwise you likely will want to turn it off. From the CLI the --dont-order-by-type option will turn this off.

https://pycqa.github.io/isort/docs/configuration/options/#dont-order-by-type
> If your project follows PEP8 or a related coding standard and has many imports this is a good default. You can turn this on from the CLI using --order-by-type.

The commit that introduced it (https://github.com/PyCQA/isort/commit/de127f5c5ea3acd6892933b0dbb4fac48f48aa73) doesn't make it clearer but I think it was a copy/paste error and it seems to me like `order-by-type` is the one really recommended for PEP8? WDYT?

Reply:
I did `--dont-order-by-type` as I preferred it always being alphabetical regardless of case.

Otherwise it will do things differently depending on the capitalization.  Which I don't like as much.

That was my personal preference.

Reply:
Ok, makes sense :)